File: Logging/MetanitLogging/FileLoggerExtensions.vb
Imports System.Runtime.CompilerServices
Imports Microsoft.Extensions.Logging

Public Module FileLoggerExtensions

    <Extension()>
    Public Function AddFile(ByVal factory As ILoggerFactory, ByVal filePath As String) As ILoggerFactory
        factory.AddProvider(New FileLoggerProvider(filePath))
        Return factory
    End Function
End Module

File: LoggingCS/MetanitLogging/FileLoggerExtensions.cs
using Microsoft.Extensions.Logging;

namespace ova.eCoin.Infrastructure.Service.Logging.Metanit
{
    public static class FileLoggerExtensions
    {
        public static ILoggerFactory AddFile(this ILoggerFactory factory, string filePath)
        {
            factory.AddProvider(new FileLoggerProvider(filePath));
            return factory;
        }
    }
}

 
